stm32-modules/heater-shaker/firmware/system/STM32F303RETx_FLASH.ld

```diff
@@ -42,7 +42,7 @@ _Min_Stack_Size = 0x400; /* required amount of stack */
 /* Specify the memory areas */
 MEMORY
 {
-FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 512K
```

Comment:
let's leave a comment here about why the change happens and when we should change it back (is it necessary now that we're not near the limit?)